module/mempool/queue/queue.go

```diff
@@ -162,23 +160,23 @@ func (q *Queue) TryAdd(element Blockify) (stored bool, new bool) {
 //     the head of _other_ cannot be a member of this queue.
 // Fails otherwise with an error. CAUTION: failing with an error leaves
 // this queue in a _dysfunctional_ (partially joined) state.
-func (q *Queue) Attach(other *Queue) error {
-	n, ok := q.Nodes[other.Head.Item.ParentID()]
-	if !ok {
-		return fmt.Errorf("other queue head does not reference known parent")
-	}
-	n.Children = append(n.Children, other.Head)
-	for identifier, node := range other.Nodes {
-		if _, ok := q.Nodes[identifier]; ok {
-			return fmt.Errorf("queues have common nodes")
-		}
-		q.Nodes[identifier] = node
-	}
-	if other.Highest.Item.Height() > q.Highest.Item.Height() {
-		q.Highest = other.Highest
-	}
-	return nil
-}
+//func (q *Queue) Attach(other *Queue) error {
```

Comment:
Agreed!

Reply:
I notice we don't use this method, but wanted to keep it around as it might get useful if we refactor EN queues, but you are right, it's always there in git